### PR TITLE
Update content.v2.json

### PR DESCRIPTION
**Pull request by crwilcox**
Retrieved Content Discovery Document from https://www.googleapis.com/discovery/v1/apis/content/v2/rest
Due to missing revision field I manually added today's date.

### CODE REVIEW

File: discoveries/content.v2.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
  "kind": "discovery#restDescription",
- "etag": "\"Zkyw9ACJZUvcYmlFaKGChzhmtnE/O1QXDlHS7167owCHVkvo0Mw47Yo\"",
+ "etag": "\"Zkyw9ACJZUvcYmlFaKGChzhmtnE/MzgF60zzzOG-2sW0R7344_YxT7w\"",
  "discoveryVersion": "v1",
  "id": "content:v2",
  "name": "content",
  "canonicalName": "Shopping Content",
  "version": "v2",
- "revision": "20180608",
+ "revision": "20180706",
```
Comment (crwilcox) [inline discoveries/content.v2.json]: I had to manually add revision as the discovery document didn't have it. I am not sure if it is reasonable to add today's date. I know there is a 20180618 version somewhere so perhaps this needs to be set to that?

@mattwhisenhunt

**Comment by dazuma** [comment]
I think I beat you to this by a few minutes https://github.com/googleapis/discovery-artifact-manager/pull/84

File: discoveries/content.v2.json (hunk 1 above)
Comment (dazuma) [reply discoveries/content.v2.json]: I suspect it should *not* be present. The revision field doesn't seem to be a required field in discovery. And it does look like there are other discovery docs (e.g. https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest) with the field missing.

**Comment by dazuma** [comment]
This is obsoleted by #84 and can be closed.

**Pull request closed without merging**